src/sb-pluginkey.c

```diff
@@ -0,0 +1,67 @@
+/**
+ *    Copyright (C) 2015 splone UG
+ *
+ *    This program is free software: you can redistribute it and/or  modify
+ *    it under the terms of the GNU Affero General Public License, version 3,
+ *    as published by the Free Software Foundation.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "sb-common.h"
+#include "rpc/sb-rpc.h"
+
+int8_t verbose_level = 0;
+
+static void print_usage(const char *name)
+{
+  LOG("Usage: %s FILE\n\n"
+      "Returns the plugin key given a plugin's public long-term key.\n",
+      name);
+}
+
+int main(int argc, char **argv)
+{
+  unsigned char clientlongtermpk[CLIENTLONGTERMPK_ARRAY_SIZE];
+  char *pluginkey = NULL;
+  FILE *fd;
+
+  if (argc <= 1) {
+    print_usage(argv[0]);
+    return (1);
+  }
+
+  fd = fopen(argv[1], "rb");
```

Comment:
would be nice if you could use the functions from `filesystem.c` like:

* `filesystem_open_read`
* `filesystem_read_all`